src/Symfony/Component/Yaml/Unescaper.php

```diff
@@ -45,9 +45,7 @@ public function unescapeSingleQuotedString(string $value): string
      */
     public function unescapeDoubleQuotedString(string $value): string
     {
-        $callback = function ($match) {
-            return $this->unescapeCharacter($match[0]);
-        };
+        $callback = fn($match) => $this->unescapeCharacter($match[0]);
```

Comment:
missing space after "fn" (in some other files too)

Reply:
I could swear I already applied fabbot diff :thinking: , done thanks.